README.md

```diff
@@ -7,3 +7,5 @@ godel-okgo-asset-nobadfuncs
 godel-okgo-asset-nobadfuncs is an asset for the gödel [okgo plugin](https://github.com/palantir/okgo). It provides the functionality of the [go-nobadfuncs](https://github.com/palantir/go-nobadfuncs) check.
 
 This check verifies that a set of blacklisted functions are not referenced.
+
+You can delegate out to the underlying check config flags with: `./godelw run-check nobadfuncs -- --print-all $file_name`
```

Comment:
Should be `$pkgPath` rather than `$file_name`

Reply:
Done